service/src/main/java/dk/dbc/search/solrdocstore/QueueFrontendAPIBean.java

```diff
@@ -49,6 +55,16 @@ public Response createQueueRule(@Context UriInfo uriInfo, String jsonContent) th
         QueueRuleEntity queueRule = jsonbContext.unmarshall(jsonContent, QueueRuleEntity.class);
         log.info("Creating queue rule: {}",queueRule.getQueue());
         entityManager.persist(queueRule);
```

Comment:
Så gerne at logik i QueueRulesBean blev brugt i stedet for at gå direkte  på entityManager